class UserExistException(Exception):
    pass
class UserNotExistException(Exception):
    pass